include/swift/SIL/SILInstruction.h

```diff
@@ -52,6 +53,9 @@ class ValueDecl;
 class VarDecl;
 class FunctionRefInst;
 
+SILValue getUndefValue(SILType Ty);
+SILValue getEmptyTypeDef(SILType Ty);
```

Comment:
I'm also not sure why we need this or what it is. If we need it, please explain. And let's not call it "TypeDef".